src/components/Home.js

```diff
@@ -0,0 +1,56 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { BsArrowRightCircle } from 'react-icons/bs';
+import { NavLink } from 'react-router-dom';
+import { cryptoThunk } from '../redux/crypto/cryptoReducer';
+
+const Home = () => {
+  const crypto = useSelector((state) => state.crypto);
+  const dispatch = useDispatch();
+
+  const crypt = crypto.crypto;
+
+  useEffect(() => {
+    dispatch(cryptoThunk());
+  }, []);
+
+  return (
+    <section className="wrapper">
+      <div className="row cryptos d-flex justify-content-center">
+        {crypt.map((coin) => (
+          <div className="currencies col-3 mb-5 p-4" key={coin.symbol} id={coin.id}>
+            <h2 className="mb-5">{coin.symbol}</h2>
+            <div className="d-flex">
+              <div>
+                <span>
+                  Count:
+                  {coin.count}
+                </span>
+                {' '}
+                <br />
+                <span>
+                  Price change:
+                  {coin.priceChange}
+                </span>
+                <br />
+                <span>
+                  Last price:
+                  {coin.lastPrice}
+                </span>
+              </div>
+              <NavLink
+                className="align-self-center arrow"
+                to="/details"
+                state={coin}
+              >
+                <BsArrowRightCircle className="details" />
+              </NavLink>
+            </div>
+          </div>
+        ))}
+      </div>
+    </section>
+  );
+};
```

Comment:
- [ ] The home page should have a search functionality that allows users to filter the items on the homepage. Kindly implement that.

  ![home](https://user-images.githubusercontent.com/36542986/189099648-c88c0c61-c1fb-485d-bb2b-c3549825a755.jpg)

- [ ] According to the provided design being followed, you should have an intro banner section that explains in a concise manner what the items on the home page represent. See design.

  ![furkan-origin-design](https://user-images.githubusercontent.com/36542986/189099981-255b6b12-9594-48dd-8722-da80362db295.jpg)

- [ ] We should stick to the design as much as possible. In the given mobile design, only two items occupy the width of the homepage, but yours is 3. Kindly fix that.

- [ ] You also have overflow issues with your items, the details spill over but your app should be responsive and the items should be contained with the item container. Kindly fix that.

- [ ] In the design we have an alternating color pattern. Kindly implement that as well.

  Your Implementation | Given Design
  :---:|:---:
  ![iit](https://user-images.githubusercontent.com/36542986/189100789-c61e8cf6-be4c-4c13-a803-966021755535.jpg) | ![jonathas-orig-home](https://user-images.githubusercontent.com/36542986/189100804-ecce8b0f-6372-4550-b415-040f3e588da4.jpg)